client-v2/src/shared/components/collectionItem/CollectionItemContainer.tsx

```diff
@@ -1,3 +1,6 @@
 import styled from 'styled-components';
 
-export default styled('div')``;
+export default styled('div')<{ isUneditable?: boolean }>`
+  pointer-events: ${({ isUneditable }) => (isUneditable ? 'none' : 'auto')};
```

Comment:
What does this pointer-events do here - do you remember? I think ideally we'd ensure that if somethings was `uneditable` then we would remove all event handlers etc. such that `pointer-events` would not be required. What would change if this CSS property was removed.

Reply:
This has zero effect that I can see so have removed it. Leftover from earlier commit - thanks for catching it!